requirements/base.txt

```diff
@@ -16,7 +16,7 @@ binaryornot==0.4.4
     # via cookiecutter
 certifi==2022.6.15
     # via requests
-chardet==4.0.0
+chardet==5.0.0
```

Comment:
No breaking change related to our code in new version https://github.com/chardet/chardet/releases/tag/5.0.0